### DISCUSSION TITLE
Center on a specific node during simulation?

### DISCUSSION THREAD

**Discussion by awaterho** [cosmograph-org/cosmos#47] [Q&A] [answered]
Hi there,
I feel this is possible with the API as it stands today, though maybe i am missing something. We have a small network centered on a central point, which of course drifts around during the simulation. 
Is there a way to "fix" the view on the central node? I feel like trackNodeById should do this, but it does not seem to work for me?

**Comment by Stukova** [comment, marked as answer]
Hi @awaterho!

The name of the `trackNodesByIds` method sounds a bit confusing, it has nothing to do with centring the camera on a Node. We've renamed it to [`trackNodePositionsByIds`](https://github.com/cosmograph-org/cosmos#track_node_positions_by_ids) for clarity.

To centre the camera on a Node, try using the [`zoomToNodeById`](https://github.com/cosmograph-org/cosmos#zoom_to_node_by_id) method together with the [`onTick`](https://github.com/cosmograph-org/cosmos#simulation_configuration) callback:

```js
simulation: {
  onTick: () => {
    graph.zoomToNodeById(nodeIdToCenter, 0, graph.getZoomLevel())
  }
}
```

Please check the example on [Codesandbox](https://codesandbox.io/s/cosmos-centering-the-node-during-simulation-zcupwt) with a demonstration. Let as know if you need any more help.

**Reply by awaterho** [reply]
Hi, yes, that's a great solution that works out of the box for our application, thanks for the help!
